language-support/java/json/src/test/scala/com/daml/lf/codegen/json/ValueConversionSpec.scala

```diff
@@ -0,0 +1,132 @@
+// Copyright (c) 2022 Digital Asset (Switzerland) GmbH and/or its affiliates. All rights reserved.
+// SPDX-License-Identifier: Apache-2.0
+
+package com.daml.lf.codegen.json
+
+import com.daml.ledger.javaapi.{data => JData}
+import com.daml.lf.data._
+import com.daml.lf.value.{Value => LfValue}
+import com.daml.lf.value.test.ValueGenerators
+import org.scalacheck.{Arbitrary, Shrink}
+import org.scalatest.Assertion
+import org.scalatest.matchers.should.Matchers
+import org.scalatest.wordspec.AnyWordSpec
+import org.scalatestplus.scalacheck.ScalaCheckPropertyChecks
+
+class ValueConversionSpec extends AnyWordSpec with Matchers with ScalaCheckPropertyChecks {
+  import LfValue._
+  import ValueGenerators._
+
+  implicit val noStringShrink: Shrink[String] = Shrink.shrinkAny[String]
+
+  implicit override val generatorDrivenConfig: PropertyCheckConfiguration =
+    PropertyCheckConfiguration(minSuccessful = 1000)
+
+  "value conversion" should {
+    "do values" in {
+      "do values" in forAll(valueGen)(testRoundTrip)
+    }
```

Comment:
This test obviates all the other cases, so you can just get rid of them.
```suggestion
    "do values" in forAll(valueGen)(testRoundTrip)
```